config/routes.rb

```diff
@@ -31,9 +35,9 @@
       get "/complete" => "subscriptions#complete", as: :subscription
     end
 
-    get "/authenticate" => "authentication#sign_in", as: :sign_in
-    post "/authenticate" => "authentication#request_sign_in_token", as: :request_sign_in_token
-    get "/authenticate/process" => "authentication#process_sign_in_token", as: :process_sign_in_token
+    # DEPRECATED: legacy routes for managing subscriptions
+    get "/authenticate" => "subscriber_authentication#sign_in"
```

Comment:
Based on our IRL chat it sounds like we'll need to keep this route long term so it might be better as a redirect.